nix/nim-status/default.nix

```diff
@@ -0,0 +1,51 @@
+{ newScope
+, xcodeWrapper
+, stdenv
+, lib }:
+let
+  inherit (lib) getAttr;
+  callPackage = newScope {};
+in rec {
+  android-x86 = callPackage ./build.nix { platform = "android"; arch = "386"; };
+  android-arm = callPackage ./build.nix { platform = "androideabi"; arch = "arm"; };
+  android-arm64 = callPackage ./build.nix { platform = "android"; arch = "arm64"; };
+
+  ios-x86 = callPackage ./build.nix { platform = "ios"; arch = "386"; };
+  ios-arm = callPackage ./build.nix { platform = "ios"; arch = "arm"; };
+  ios-arm64 = callPackage ./build.nix { platform = "ios"; arch = "arm64"; };
+
+  android = stdenv.mkDerivation {
+    name = "nim-status.android";
+    phases = [ "symlinkPhase" ];
+    symlinkPhase = ''
```

Comment:
Same here, you can just use a `builder = writeScript "nim-status-android-builder" ''`.

Reply:
Done.